ui/src/config/constants.tsx

```diff
@@ -5,3 +5,4 @@ window.config = window.config || {
 
 export const API_URL = window.config.API_URL;
 export const GH_CLIENT_ID = window.config.GH_CLIENT_ID;
+export const API_VERSION = 'v1';
```

Comment:
I think we should not hard code this, because once we have `v2` then again we'll have to change here, could you please read this through `window.config` as well ??

Reply:
I guess we can keep this hard-coded so that we can make sure that UI is compatible whenever we have new version of API available otherwise there can be chances that UI might break

Reply:
With adding it in `window.config` you can still get to know if it might break the ui or not

Reply:
In short even if we get it through `window.config` we can make sure that whethere it is compatible with UI or not